src/clj/game/core/commands.clj

```diff
@@ -114,7 +114,7 @@
 
 (defn rez-all
   [state side eid cards]
-  (let [c (first cards)]
+  (when-let [c (first cards)]
```

Comment:
2) Change this to `(if-let [c (first cards)] (wait-for ...) (effect-completed state side eid))`